modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/index.js

```diff
@@ -0,0 +1,436 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClaySticker from '@clayui/sticker';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import {useEffect, useMemo, useState} from 'react';
+import {IconButton} from '../../../../common/components';
+import RoundedGroupButtons from '../../../../common/components/RoundedGroupButtons';
+import Table from '../../../../common/components/Table';
+import {useApplicationProvider} from '../../../../common/context/AppPropertiesProvider';
+import {getActivationLicenseKey} from '../../../../common/services/liferay/rest/raysource/LicenseKeys';
+import getCurrentEndDate from '../../../../common/utils/getCurrentEndDate';
+import {useCustomerPortal} from '../../context';
+import {getPascalCase} from '../../utils/getPascalCase';
+
+const ACTIVATION_KEYS_LICENSE_FILTER_TYPES = {
+	active: ({expirationDate, startDate}) =>
+		new Date(startDate) < new Date() &&
+		new Date(expirationDate) > new Date(),
+	expired: ({expirationDate}) => new Date(expirationDate) < new Date(),
+	notActivated: ({startDate}) => new Date(startDate) > new Date(),
+};
+
+const ACTIVATION_STATUS = {
+	active: {
+		color: 'success',
+		id: 'active',
+		title: 'Active',
+	},
+	all: {
+		color: 'none',
+		id: 'all',
+		title: 'All',
+	},
+	expired: {
+		color: 'danger',
+		id: 'expired',
+		title: 'Expired',
+	},
+	notActivated: {
+		color: 'info',
+		id: 'notActivated',
+		title: 'Not Activated',
+	},
+};
```

Comment:
Would be this constant on the constants directory?

Reply:
Hi @igordmf. I think this is not necessary because this variable is only being used in this component

Reply:
Yes @iagochaves, I agree!